easybuild/easyconfigs/m/MVAPICH2/MVAPICH2-1.7-GCC-4.6.3-hwloc-chkpt.eb

```diff
@@ -12,13 +12,11 @@ sources = [SOURCELOWER_TGZ]
 # note: this URL will only work for the most recent version (previous versions no longer available?)
 source_urls = ['http://mvapich.cse.ohio-state.edu/download/mvapich2']
 
-rdma_type = "gen2"  # 'gen2' or 'udapl'
-
 # enable building of MPE routines
-withmpe = True
+configopts = '--with-mpe '
 
 # enable hwloc support
-withhwloc = True
+configopts += '--with-hwloc '
```

Comment:
I would just leave this as is, since we shouldn't remove `withmpe` and `withhwloc` from the easyblock?